src/tests/user-actions.test.js

```diff
@@ -0,0 +1,62 @@
+import {
+    login,
+    LOGIN_ERROR,
+    LOGIN_PENDING,
+    LOGIN_SUCCESS,
+} from '../store/user/actions';
+import { loginApi } from '../api';
+import { enableFetchMocks } from 'jest-fetch-mock';
+
+enableFetchMocks();
+
+describe('userActions', () => {
+    beforeEach(() => {
+        fetch.resetMocks();
+    });
+    it('Login successful, creates LOGIN_PENDING and LOGIN_SUCCESS', async () => {
```

Comment:
usually tests start with 
it('Should do/return/etc ...

Reply:
done